src/sideof.jl

```diff
@@ -58,6 +58,7 @@ Possible results are `IN`, `OUT` or `ON` the `ring`.
 """
 function sideof(point::Point, ring::Ring)
   assertion(CoordRefSystems.ncoords(crs(point)) == 2, "points must have 2 coordinates")
+  point = Proj(crs(ring))(point)
```

Comment:
```suggestion
  point′ = point |> Proj(crs(ring))
```

we don't usually overwrite variables when the type changes because that leads to type instability